### PR TITLE
Update download urls, remove windows support and update default fdb version

### LINKED ISSUE #1: Use Github releases to download artifacts
The FDB community removed downloads from the official website, and moved them to Github Releases. We need to change the Action to be compatible with this new repo.

### LINKED ISSUE #2: Remove Windows from list of supported platform
Windows is no more an official FDB Build. We should remove it

### LINKED ISSUE #3: Bump default to 6.3
6.2 is now unsupported by the FDB community, we need to bump it to 6.3

### PR DESCRIPTION
**Pull request by khouari1**
- Fixes #1
- Fixes #2
- Fixes #3

I made some updates to package.json. Let me know if if I should change any of that. I guess we will need to update the version at some point. Not sure if I should do that in this PR or we can do in another.

### CODE REVIEW

File: index.js
Comment (PierreZ) [inline index.js:26]: I'm wondering if we should fail on that case :thinking: Users can migrate from Clikengo to this Action, and might expect Windows's support.

**Review by PierreZ — CHANGES_REQUESTED** [review]
Thanks for the PR :tada: 

I left a small comment on Windows's support. 

Once the change has been made, I'll wait for @speedy37's approval before merging, as he's the original author.

File: index.js
Comment (khouari1) [reply index.js:26]: Yes, makes sense. I guess we could have a default case which logs a message saying that the os is unsupported and throws an error, then that error would get caught by try-catch and the action would fail?

Comment (PierreZ) [reply index.js:26]: That sounds perfect :+1:

**Review by PierreZ — APPROVED** [review]

**Pull request merged**
